e2e/builder/global_test.go

```diff
@@ -48,8 +49,15 @@ func TestRunGlobalInstall(t *testing.T) {
 		}
 	}
 
-	WithNewTestNamespace(t, func(ns string) {
-		Expect(Kamel("install", "-n", ns, "--global").Execute()).To(Succeed())
+	WithNewTestNamespace(t, func(operatorNamespace string) {
+		Expect(Kamel("install", "-n", operatorNamespace, "--global").Execute()).To(Succeed())
```

Comment:
Shouldn't it be called after the namespace is overwritten for OCP, and not before?

Reply:
The operator is installed to the `openshift-operators` anyway. That means that the `-n` + `--global` combo does not work together but you are right. Setting the namespace **before** can rise the test readability.

Reply:
Right, it's more that it's not the case without OLM, i.e., `kamel install --global --olm=false`, so the tests would also work in that case.

Reply:
oh, I totally forget about the non-OLM installation. Thank you for spotting it.

Reply:
Done